lib/views/sessions/new.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Please sign in!</h2>
+
+<form action="/sessions" method="post" id="submit-forms">
+	Username: <input type="text" name="username" id="entry-box"><br>
```

Comment:
Also, don't use `<br />` unless you're using it structurally (i.e. certain things like poems that need to be broken up in a particular way). If you want a thing to have a line break after it, give that thing a CSS `display: block;`.